Chaincase.UI/wwwroot/global.js

```diff
@@ -0,0 +1,19 @@
+function dispatchEventWrapper(evt) {
+  window.dispatchEvent(new CustomEvent(evt))
+}
+
+if (!navigator.share && navigator.clipboard) {
+  navigator.share = (data) => {
+    const toast = document.createElement('ion-toast');
+    toast.message = data.title ? `${data.title} copied to clipboard` : 'Copied to clipboard';
+    toast.duration = 2000;
+    navigator.clipboard.writeText(data.text || data.url).catch(reason => console.error(reason));
+    document.body.appendChild(toast);
+    return toast.present();
```

Comment:
@armins88 I know you were looking for an example of using toast and this isn't exactly one with a button or in c#/razor, but it is an example

Reply:
https://github.com/Kukks/Blazor.Ionic/blob/3b0c1a72f13ce426d914c0a5bad1cbf2f347ce92/Blazor.Ionic.Demo.UI/Pages/Demo.razor#L55

Reply:
https://github.com/Kukks/Blazor.Ionic/blob/3b0c1a72f13ce426d914c0a5bad1cbf2f347ce92/Blazor.Ionic.Demo.UI/Pages/Demo.razor#L55